source/Octopus.Shared/Octopus.Shared.csproj

```diff
@@ -0,0 +1,114 @@
+    <PackageReference Include="Octopus.Configuration" Version="4.1.0" />
+    <PackageReference Include="Octopus.Diagnostics" Version="2.1.0" />
+    <PackageReference Include="Octopus.Time" Version="1.1.339" />
+    <PackageReference Include="Polly" Version="5.3.1" />
+    <PackageReference Include="Portable.BouncyCastle" Version="1.8.4" />
+    <PackageReference Include="System.Management" Version="4.7.0" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    <PackageReference Include="TaskScheduler" Version="2.7.2" />
+    <PackageReference Include="Octopus.CoreUtilities" Version="2.1.143" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETCoreApp' ">
+    <PackageReference Include="Microsoft.DotNet.Analyzers.Compatibility" Version="0.2.12-alpha">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="1.1.0" />
+    <PackageReference Include="System.ServiceModel.Primitives" Version="4.5.3" />
+    <PackageReference Include="System.ServiceProcess.ServiceController" Version="4.5.0" />
+    <PackageReference Include="System.IO.FileSystem.AccessControl" Version="4.5.0" />
+    <PackageReference Include="Mono.Posix.NETStandard" Version="1.0.0" />
+    <PackageReference Include="NLog" Version="4.7.6" />
+    <PackageReference Include="Nito.AsyncEx" Version="5.0.0" />
+    <PackageReference Include="System.Diagnostics.EventLog" Version="4.5.0" />
+    <PackageReference Include="System.Text.Encoding.CodePages" Version="4.5.1" />
+    <PackageReference Include="System.DirectoryServices.AccountManagement" Version="4.5.0" />
+    <PackageReference Include="System.Security.Cryptography.ProtectedData" Version="4.5.0" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">
+    <PackageReference Include="NLog" Version="4.0.1" />
+    <PackageReference Include="Nito.AsyncEx" Version="3.0.1" />
+    <PackageReference Include="System.Runtime" Version="4.3.0" />
+    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
+    <Reference Include="System.Security" />
+    <Reference Include="System.DirectoryServices" />
+    <Reference Include="System.IdentityModel" />
+    <Reference Include="System.ServiceProcess" />
+    <Reference Include="System.Web" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Internals\Options\_Readme.txt" />
+    <EmbeddedResource Include="Startup\PathsToDeleteOnStartup.core.txt" />
+    <EmbeddedResource Include="Startup\PathsToDeleteOnStartup.netfx.txt" />
+    <Compile Update="Startup\WindowsServiceAdapter.cs">
+      <SubType>Component</SubType>
+    </Compile>
+    <None Include="icon.png" Pack="true" PackagePath="" />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Messages\Deploy\Steps" />
```

Comment:
This folder appears to be missing for me when I load this locally. Is this a problem? I'm guessing its just a section of the csproj that shouldn't have been there to begin with.